src/main/java/network/brightspots/rcv/CommonDataFormatReader.java

```diff
@@ -363,6 +363,10 @@ void parseJson(List<CastVoteRecord> castVoteRecords)
       for (Object cvrContestSelectionObject : cvrContestSelections) {
         HashMap cvrContestSelection = (HashMap) cvrContestSelectionObject;
         String contestSelectionId = (String) cvrContestSelection.get("ContestSelectionId");
+        if (!contestSelections.containsKey(contestSelectionId)) {
+          Logger.severe("ContestSelection \"%s\" from CVR not found!", contestSelectionId);
```

Comment:
Will the user know what a "ContestSelection" is? Cause I don't...

Reply:
They "should".  ContestSelection is CDF-speak for "one of the things you can pick for a particular contest".  I think ClearVote calls it "option" and more often we just say "candidate" because that's the typical domain.  CDF can also describe ballot measure selections, retention contest selections, and party contest selections.  It is truly a kitchen sink format.